security/rbac/scripts/enable-rbac-rest-proxy.sh

```diff
@@ -94,14 +95,18 @@ else
   echo "FAIL: Something went wrong, check output"
 fi
 
-echo -e "\n# Grant the principal User:$CLIENTB to the ResourceOwner role for Group:$CONSUMER_GROUP"
-echo "confluent iam rolebinding create --principal User:$CLIENTB --role ResourceOwner --resource Group:$CONSUMER_GROUP --kafka-cluster-id $KAFKA_CLUSTER_ID"
-confluent iam rolebinding create --principal User:$CLIENTB --role ResourceOwner --resource Group:$CONSUMER_GROUP --kafka-cluster-id $KAFKA_CLUSTER_ID
+echo -e "\n# Grant the principal User:$CLIENTB to the DeveloperRead role for Group:$CONSUMER_GROUP"
+echo "confluent iam rolebinding create --principal User:$CLIENTB --role DeveloperRead --resource Group:$CONSUMER_GROUP --kafka-cluster-id $KAFKA_CLUSTER_ID"
+confluent iam rolebinding create --principal User:$CLIENTB --role DeveloperRead --resource Group:$CONSUMER_GROUP --kafka-cluster-id $KAFKA_CLUSTER_ID
 
 echo -e "\n# Consume messages from the topic $TOPIC, after authorization (should pass)"
 echo 'curl -u clientb:clientb1 --silent -X GET -H "Accept: application/vnd.kafka.json.v2+json" http://localhost:8082/consumers/'"$CONSUMER_GROUP"'/instances/my_consumer_instance/records'
 curl -u $CLIENTB:clientb1 --silent -X GET -H "Accept: application/vnd.kafka.json.v2+json" http://localhost:8082/consumers/$CONSUMER_GROUP/instances/my_consumer_instance/records
+echo
```

Comment:
Is this intentional?

Reply:
@rnpridgeon Yes, it forces a new line

Reply:
yeah I figured that's what you were going for but wasn't sure since the subsequent echo is prefixed with `\n`. Anyway LGTM

Reply:
#curllife